Felienne/hedy
# [BUG FIXES] Fixes faulty re-direct and makes sure achievement always in user view

**Description**
In this PR we fix two small bugs:

- When deleting your public-profile you would be re-directed to /programs instead of /my-profile
- Achievements where always shown 60% from top instead of from user view -> Making them pop-up not in view

**Fix for**
This PR fixes #1830 and fixes #1831.

**How to test**
Delete your public profile and notice the correct re-direct. Get an achievement on a long page and notice the pop-up is always in view.
